Jenkinsfile.ci

```diff
@@ -0,0 +1,100 @@
+#!groovy
+
+try {
+    testing()
+} catch (err) {
+    currentBuild.result = "FAILED"
+    throw err
+}
+
+def testing() {
+    stage('Testing') {
+        parallel([
+                'ubuntu-test' : { ubuntuTesting() },
+                'windows-test': { windowsTesting() }
+        ])
+    }
+}
+
+def getUserUid() {
```

Comment:
Order of function in the file is incorrect. Should be:

top-level-function1
top-level-function2
middle-level-function1
low-level-function1